clang/lib/Sema/SemaOpenMP.cpp

```diff
@@ -23094,6 +23094,50 @@ void Sema::checkDeclIsAllowedInOpenMPTarget(Expr *E, Decl *D,
   checkDeclInTargetContext(E->getExprLoc(), E->getSourceRange(), *this, D);
 }
 
+Decl *AddOMPDeclareTargetDeclAttr(const DeclRefExpr *DeclRef,
+                                  Decl *TargetDecl) {
+  if (DeclRef) {
+    Decl *DeclVar = (Decl *)DeclRef->getDecl();
+    DeclVar->addAttr(TargetDecl->getAttr<OMPDeclareTargetDeclAttr>());
+    return DeclVar;
+  } else
+    return nullptr;
+}
```

Comment:
```suggestion
  Decl *DeclVar = nullptr;
  if (DeclRef) {
    DeclVar = (Decl *)DeclRef->getDecl();
    DeclVar->addAttr(TargetDecl->getAttr<OMPDeclareTargetDeclAttr>());
  }
  return DeclVar;
```

Also, correct the alignment.